docs-ref-services/service-map.md

```diff
@@ -16,17 +16,25 @@ ms.service: Service Map
 # Azure Service Map packages for Node.js

```

Comment:
Since it's only one package should this read `Azure Service Map package for Node.js` for now?